ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2830,7 +2830,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     }
                     break;
                 case StrategyTradeWorkFlowState.GoShortPlaceProfitTargetsPending:
-                    if (IsHistoricalTradeOrPlayBack || !IsSubmitTargetsAndConfirm)

```

Comment:
This is required - IsHistoricalTradeOrPlayBack  fastest route no confrmation process - only for realtime modes or unsafe modes